src/docs/development/platform-integration/platform-channels.md

```diff
@@ -4,13 +4,15 @@ short-title: Platform-specific code
 description: Learn how to write custom platform-specific code in your app.
 ---
 
+[PENDING: Is this page up-to-date?]
```

Comment:
This is still interesting, particularly the part about data serialization and supported types. I think this is still current and valid for all "native" plugins. Only "web in a federated architecture" doesn't use the platform channels.

Reply:
Thanks!